diff --git a/src/loader/pci_id_driver_map.c b/src/loader/pci_id_driver_map.c
index 8b2079e4354..c06c2ce8561 100644
--- a/src/loader/pci_id_driver_map.c
+++ b/src/loader/pci_id_driver_map.c
@@ -27,7 +27,14 @@ int is_nouveau_vieux(int fd);
 
 #include <stdlib.h>
 #include <xf86drm.h>
-#include <nouveau_drm.h>
+/* #include <nouveau_drm.h> */
+#define NOUVEAU_GETPARAM_CHIPSET_ID      11
+#define DRM_NOUVEAU_GETPARAM           0x00
+
+struct drm_nouveau_getparam {
+        uint64_t param;
+        uint64_t value;
+};
 
 static int
 nouveau_chipset(int fd)
