From ba558294d9d0c8f003737134a195d2e059b3e3db Mon Sep 17 00:00:00 2001
From: Felipe Borges <felipeborges@gnome.org>
Date: Wed, 3 Jun 2020 15:32:01 +0200
Subject: [PATCH] WIP

---
 meson.build | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/meson.build b/meson.build
index 405c589..5725b7a 100644
--- a/meson.build
+++ b/meson.build
@@ -346,6 +346,8 @@ if d.found()
   spice_gtk_config_data.set('USE_SMARTCARD', '1')
 endif
 
+spice_glib_deps += compiler.find_library('m', required: false)
+
 #
 # global C defines
 #
-- 
2.26.2

